app/bundles/LeadBundle/Views/Lead/lead.html.php

```diff
@@ -597,6 +597,21 @@ class="fa fa-check <?php echo (1 == $company['is_primary']) ? 'primary' : ''; ?>
             <?php endforeach; ?>
             <div class="clearfix"></div>
         </div>
+        <div class="pa-sm panel-segments">
+          <div class="panel-title">
+            <?php echo $view['translator']->trans('mautic.lead.lead.lists'); ?>
+          </div>
+          <?php foreach ($lists[$lead->getId()] as $key => $list): ?>
+            <h5 class="pull-left mt-xs mr-xs">
+                  <span class="label label-success">
+                      <a href="<?php echo $view['router']->path('mautic_segment_action', ['objectAction' => 'view', 'objectId' => $list['id']]); ?>" style="color: white;">
```

Comment:
If you add data attribute  `data-toggle="ajax"` to anchor, links will loaded with ajax (same in companies).
Can you do that?

Reply:
Great point. I've added it to both links.